src/patterns/variable_decl.rs

```diff
@@ -18,7 +17,6 @@ impl Pattern for VariableDeclPattern {
 	(var_declaration
     	(var_spec
         	name: (identifier) @name
-            type: (type_identifier) @type
```

Comment:
Since there are cases where there is a type, wouldn't it be better to make it optional?